src/System.Security.Cryptography.Primitives/src/facade/System.Security.Cryptography.Primitives.csproj

```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <!-- Setting default TargetGroup before importing dir.prop -->
-    <TargetGroup Condition="'$(TargetGroup)' == ''">net46</TargetGroup>
+    <Configuration Condition="'$(Configuration)'==''">net46_Debug</Configuration>
```

Comment:
Why would we want net46 to be the default?  That wouldn't have any code in it (if VS and plugins understood our conditional source directives)